src/typeInference/annotateBinaryOperator.ts

```diff
@@ -29,12 +29,10 @@ export function annotateBinOp(
 ): SequenceType {
 	// If we don't have the left and right type, we cannot infer the current type
 	if (!left || !right) {
-		const itemReturn = {
+		return {
```

Comment:
Didn't martin say that binary operator always returns an xs:numeric?

Reply:
Good point, will change it

Reply:
This should then also be EXACTLY_ONE right?

Reply:
Yeah I think exactly_one would be best